src/vat/remotes.js

```diff
@@ -308,9 +352,18 @@ export function makeRemoteManager(
     // todo: now send an ack
   }
 
+  const dl = makeDecisionList(
+    myVatID,
+    isLeader,
+    followers,
+    getReadyMessages,
+    deliver,
+    sendDecisionTo,
+  );
+
   function commsReceived(fromHostID, wireMessage) {
     // console.log(`commsReceived ${fromHostID}, ${wireMessage}`);
-    const hr = getHostRemote(fromHostID);
+    getHostRemote(fromHostID);
```

Comment:
let's add a "TODO: do we need the side-effects of this" comment here. I can't immediately remember if I was calling this to use `hr` (and then deleted some code that used it), or if we're depending upon creating a HostRemote upon receipt of a message. Removing the `const hr =` makes it look like we're specifically calling it for the side-effect, which might not be the case.